docs/assets/virtual-select.js

```diff
@@ -1925,12 +1922,14 @@ var VirtualSelect = /*#__PURE__*/function () {
         visibleOptions = [newOption].concat(virtual_select_toConsumableArray(visibleOptions));
       }
       this.visibleOptions = visibleOptions;
+      // update number of visible options
+      this.visibleOptionsCount = visibleOptions.length;
       this.renderOptions();
     }
   }, {
     key: "setOptionsPosition",
     value: function setOptionsPosition(startIndex) {
-      var top = (startIndex || this.getVisibleStartIndex()) * this.optionHeight;
+      var top = parseInt((startIndex || this.getVisibleStartIndex()) * this.optionHeight);
```

Comment:
Some comments would be relevant for future understanding

Reply:
@rugoncalves will do that. Thanks

Reply:
@rugoncalves done ✅